repositories/experiment.repository.js

```diff
@@ -43,6 +66,14 @@ module.exports = new (class ExperimentsRepository extends MongoStorage {
   }
 
   async updateExperimentStatus(id, newStatus) {
-    return await this.update(id, { status: newStatus });
+    return await this.update(id, { status: newStatus }).populate({path: 'goals'});
+  }
+
+  async addGoal(experimentId, goalId){
+    return await this.update(experimentId, {$push: {goals: goalId}}).populate({path: 'goals'});
+  }
+  async removeGoal(experimentId, goalId){
+    return await this.update(experimentId, {$pull: {goals: goalId}}).populate({path: 'goals'});
   }
+
 })();
```

Comment:
Great update on the experiment repository!

Reply:
Thanks :)